samples/custom-consent-authorizationserver/samples-custom-consent-authorizationserver.gradle

```diff
@@ -8,6 +8,10 @@ group = project.rootProject.group
 version = project.rootProject.version
 sourceCompatibility = "17"
 
+test {
```

Comment:
Please revert all changes in `custom-consent-authorizationserver` since we have test coverage in `default-authorizationserver` and there is no need to duplicate